explore/visualization_vega_lite_altair/heatmap_vega-lite.json

```diff
@@ -0,0 +1,37 @@
+{
+  "config": {
+    "scale": {
+      "textBandWidth": 30,
+      "bandSize": 15
+    },
+    "cell": {
+      "height": 350,
+      "width": 500
+    },
+    "mark": {
+      "applyColorToBackground": true
+    }
+  },
+  "data": {
+    "url": "./heatmap_data_Altair_compatible.json"
```

Comment:
Can we change the name of `heatmap_data_Altair_compatible.json` to `heatmap-data.json`?